docs/contributor/draft/rate-limit-api.md

```diff
@@ -0,0 +1,122 @@
+| wokloadSelector                                  | object      | in case of empty, it's applied for the whole namespace          | no                            |
+| workloadSelector.labels                          | object      | allows to select given workload by a selector                   | no                            |
+| localRateLimit                                   | object      | allows to describe local rate limit properties                  | yes                           |
+| localRateLimit.by                                | object      | container for criteria you can limit by (for the future)        | yes                           |
+| localRateLimit.by.requests                       | object      | allows to set rate limit based on plain number of requests      | yes                           |
+| localRateLimit.by.requests.time                  | object      | allows to describe local rate limiting using time periods       | yes (if buckets not defined)  |
+| localRateLimit.by.requests.buckets               | object      | allows to describe local rate limiting using buckets            | yes (if time not defined)     |
+| localRateLimit.by.requests.time.unit             | string      | describes unit of time for rate limit                           | yes                           |
+| localRateLimit.by.requests.time.requestsLimit    | int         |describes requests limit for a declared time unit                | yes                           |
+| localRateLimit.by.requests.buckets.maxTokens     | int         |describes max amount of tokens that can stack up                 | yes                           |
+| localRateLimit.by.requests.buckets.tokensPerFill | int         | describes how many tokens to fill every defined period of time  | yes                           |
+| localRateLimit.by.requests.buckets.fillInterval  | time        |  describes how often fill tokens                                | yes                           |
+
+## Usage example
+
+```yaml
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: RateLimit
+metadata:
+  name: httpbin-local-rate-limit
+  namespace: default
+spec:
+  workloadSelector:
+    labels:
+      app: httpbin
+  localRateLimit:
+    by:
+      requests:
+        time: # exclusive with buckets, got to choose one
+          unit: "second" 
+          requestsLimit: 30
+        buckets:
+          max_tokens: 1000
+          tokens_per_fill: 1000
+          fill_interval: 1s
+
+---
+
+# eventually we could extend it later into such direction
+
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: RateLimit
+metadata:
+  name: default
+  namespace: kyma-system
+spec:
+  workloadSelector:
+    labels:
+      app: httpbin
+  localRateLimit: # separating it like this allows to extend fields with new use cases. Also every field can be extended with it's specifics
+    localRateLimit:
```

Comment:
The field name is doubled here (`localRateLimit.localRateLimit`)

Reply:
oh thanks